componentes/App.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { dataPost } from "../utils/data-post";
```

Comment:
essa variavel pra fazer sentido seria 
```suggestion
import { postsData } from "../utils/data-post";
```

Reply:
ai tem que mudar o nome do arquivo tbm